src/LibPQ.jl

```diff
@@ -13,6 +13,8 @@ using DocStringExtensions
 using Decimals
 using FileWatching
 using Tables
+using Infinity
```

Comment:
I believe this is all that needed:
```suggestion
using Infinity: InfExtendedTime, isposinf
```

Reply:
Looks like you're right, except we also need to add `∞` as well.